.github/renovate.json5

```diff
@@ -13,6 +13,20 @@
       platformAutomerge: true,
     },
 
+    // Special rule for Iceberg and related artifacts
+    {
+      matchManagers: ["maven", "gradle"],
+      matchUpdateTypes: ["minor", "patch"],
+      matchPackagePatterns: [
+        "^org[.]apache[.]hadoop:hadoop-aws$",
+        "^org[.]apache[.]hadoop:hadoop-common$",
+        "^org[.]apache[.]iceberg:.*",
+        "^software[.]amazon[.]awssdk:.*",
+      ],
```

Comment:
Should we also disable auto merge for hive? 

we have two runtime dependencies of hive which should be in mapping with the Iceberg. 

https://github.com/projectnessie/iceberg-catalog-migrator/blob/1ea1ddc42875133cb41a3b3a8e06ecba0b6de534/cli/build.gradle.kts#L49-L77